app/models/resource.rb

```diff
@@ -1,10 +1,11 @@
 class Resource < ActiveRecord::Base
   belongs_to :conference
+  validates :name, :quantity, presence: true
   validate :used_less_than_quantity
```

Comment:
Just to be in sync with what we are actually validating, we can perhaps rephrase this into something like `used_no_more_than_quantity`